examples/ruby/spec/interactions/frames_spec.rb

```diff
@@ -3,5 +3,40 @@
 require 'spec_helper'
 
 RSpec.describe 'Frames' do
+  #set session
   let(:driver) { start_session }
+  let(:wait) { Selenium::WebDriver::Wait.new(timeout: 2) }
+
+  it 'interacts with elements inside iframes' do
+    #navigate to web page
```

Comment:
The code comments might trip up IDE's as generally they have one space infront of the `#`